glide.lock

```diff
@@ -302,13 +304,13 @@ imports:
 - name: github.com/robfig/glock
   version: 39b969c322811a58be9ec8be9d65198d43d8ba82
 - name: github.com/rubyist/circuitbreaker
-  version: af9583088c3453556f6e4f580693ba8606ca459a
+  version: 9d2405c9a39876d9c85a8d6cc8ce16b63eb83610
```

Comment:
Huh? Usually, when diligently updating dependencies, I audit the changes to spot additions that we should obviously be taking advantage of, such as this one.

Perhaps you could do that as well, though in a new PR if you wish, since you're taking on this burden.